SU-SWS/stanford_basic
# updated package.json and gruntfile.js with latest and compiled code

# READY FOR REVIEW

# Summary
- Updated package.json and Gruntfile.js to mirror Decanter (specifically for version of Bourbon being used)
- Compiled code

# Needed By (Date)
- As soon as possible (for release of Arch)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Ensure that aforementioned files are updated and that related grunt commands are working (sass and decanter specifically) without error

# Affected Projects or Products
- stanford_basic
- Archaeology
- suhumsci (theme)

# Associated Issues and/or People
- N/A

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
